metadata.rb

```diff
@@ -8,5 +8,6 @@
 description      'Installs/Configures yum-osuosl'
 version          '2.0.0'
 
-supports 'centos', '~> 7.0'
-supports 'centos', '~> 8.0'
+supports         'almalinux', '~> 8.0'
+supports         'centos', '~> 7.0'
+supports         'centos', '~> 8.0'
```

Comment:
```suggestion
supports         'centos_stream', '~> 8.0'
```

Reply:
> 

Added the changes for centos_stream support